site/src/api/api.ts

```diff
@@ -1455,8 +1455,10 @@ class ApiMethods {
     return response.data;
   };
 
-  getGroup = async (groupId: string): Promise<TypesGen.Group> => {
-    const response = await this.axios.get(`/api/v2/groups/${groupId}`);
+  getGroup = async (groupName: string): Promise<TypesGen.Group> => {
+    const response = await this.axios.get(
+      `/api/v2/organizations/default/groups/${groupName}`,
```

Comment:
@kylecarbs is this what you mean?

Reply:
Yeah, did this route already work with a group name? I think I may be wrong here

Reply:
yup, this route accepts a name or an id.